src/main/java/in/zapr/druid/druidry/client/DruidConfiguration.java

```diff
@@ -0,0 +1,44 @@
+package in.zapr.druid.druidry.client;
+
+import lombok.Builder;
+import lombok.Getter;
+import lombok.Setter;
+
+@Builder
+@Getter
+@Setter
+public class DruidConfiguration {
```

Comment:
Do we really need this class? Can we take org.glassfish.jersey.client.ClientConfig as a parameter to DruidJerseyClient? The ClientConfig and Configuration classes have lot of methods to add custom headers, timeout, ExecutorService and so on.